src/EditorFeatures/Core/Preview/AbstractPreviewFactoryService.cs

```diff
@@ -58,7 +58,6 @@ internal abstract class AbstractPreviewFactoryService<TDifferenceViewer> : IPrev
             ITextViewRoleSet previewRoleSet,
             IGlobalOptionService globalOptions)
         {
-            threadingContext.ThrowIfNotOnUIThread();
```

Comment:
Why this change?
Currently all preview window integration tests are failing because they are hitting this assert (this is not being called from the UI thread).  I removed the assert as we didn't appear to need it for this ctor.

Why are we hitting this only in this PR?
1.  [Cyrus merged a change in main-vs-deps](https://github.com/dotnet/roslyn/pull/60526/files#diff-ef037b64a053faeb04316eba23171bfbded587d3abfcb6935d8149f8a34f572cR61) which changed the behavior here to throw if not on the UI thread when previously it only asserted a UI thread existed (per Sam's comment [here](https://github.com/dotnet/roslyn/pull/60526#discussion_r840903373))
2.  This passed all PR checks originally because it was being called on the UI thread in main-vs-deps.
3.  However, a [recent change in main](https://github.com/dotnet/roslyn/commit/494ef047503e729fda923542c13c978e303ee755#diff-53cfc7ea1a75becd23ef32bc8ba79795fb78ce960066638b585a8d32b1565322R194) modified this to no longer be called on the UI thread.
4.  So now when we merge, the assert throws because its no longer on the UI thread.  Fun!